cmd/cli/exec/exec.go

```diff
@@ -210,9 +314,12 @@ func addInlineContent(ctx context.Context, codeLocation string, job *models.Job)
 
 	target := "/code"
 
-	finfo, _ := os.Stat(absPath)
-	if !finfo.IsDir() {
-		target = fmt.Sprintf("/code/%s", finfo.Name())
+	if finfo, err := os.Stat(absPath); err != nil {
+		return fmt.Errorf("file '%s' not found", codeLocation)
```

Comment:
might be worth include the specific error returned here (unless this is being passed directly to the user) stat can fail for other reasons, although usually due to file not being found of course.